moneo.py

```diff
@@ -4,97 +4,115 @@
         parser.print_helper()
         exit(1)
 
-if __name__ == '__main__':
 
-    parser = argparse.ArgumentParser(description='Moneo CLI',prog='moneo.py',usage='%(prog)s [-d ] [-c HOST_INI] [{manager,workers,full}] \
-    \nusage: %(prog)s [-s ] [-c HOST_INI] [{manager,workers,full}] \
-    \nusage: %(prog)s [-j JOB_ID ] [-c HOST_INI] \
-    \ni.e. python3 moneo.py -d -c ./host.ini full')  
-
-    parser.add_argument('-c','--host_ini',default='./host.ini', help='Provide filepath and name of ansible config file. The default is host.ini in the Moneo directory.' )    
-    parser.add_argument('-j','--job_id',type=str, help='Job ID for filtering metrics by job group. Host.ini file required. Cannot be specified during deployment and shutdown' )
-    parser.add_argument('-d','--deploy', action='store_true',help='Requires config file to be specified (i.e. -c host.ini) or file to be in Moneo directory.')
-    parser.add_argument('-s','--shutdown',action='store_true', help='Requires config file to be specified (i.e. -c host.ini) or file to be in Moneo directory.')
-    parser.add_argument('-i', '--insights',action='store_true', help='Enable exporting of metrics to Azure Insights. Requires a valid instrumentation key and base_url for the Prometheus DB in config.ini')
-    parser.add_argument('type', metavar='type', type=str,default=['full'], nargs="*", help='Type of deployment/shutdown. Choices: {manager,workers,full}. Default: full.')    
+if __name__ == '__main__':
+    #   parser options
+    parser = argparse.ArgumentParser(description='Moneo CLI Help Menu', prog='moneo.py', usage='%(prog)s [-d ] [-c HOST_INI] [{manager,workers,full}] \
+        \nusage: %(prog)s [-s ] [-c HOST_INI] [{manager,workers,full}] \
+        \nusage: %(prog)s [-j JOB_ID ] [-c HOST_INI] \
+        \ni.e. python3 moneo.py -d -c ./host.ini full')
 
+    parser.add_argument('-c', '--host_ini', default='./host.ini', help='Provide filepath and name of ansible config file. The default is host.ini in the Moneo directory.')
+    parser.add_argument('-j', '--job_id', type=str, help='Job ID for filtering metrics by job group. Host.ini file required. Cannot be specified during deployment and shutdown')
+    parser.add_argument('-d', '--deploy', action='store_true', help='Requires config file to be specified (i.e. -c host.ini) or file to be in Moneo directory.')
+    parser.add_argument('-s', '--shutdown', action='store_true', help='Requires config file to be specified (i.e. -c host.ini) or file to be in Moneo directory.')
+    parser.add_argument('-i', '--insights', action='store_true', help='Experimental feature: Enable exporting of metrics to Azure Insights. Requires a valid instrumentation key and base_url for the Prometheus DB in config.ini')
+    parser.add_argument('type', metavar='type', type=str, default=['full'], nargs="*", help='Type of deployment/shutdown. Choices: {manager,workers,full}. Default: full.')
```

Comment:
So it's possible to have multiple types, like `manager workers`?

Reply:
It seems like it just grabs the first argument. I think what we want is that it errors out with 2 arguments. I will address this.

Reply:
Addressed